### DISCUSSION TITLE
Link addresses to companies

### DISCUSSION THREAD

**Discussion by fxmb** [medusajs/b2b-starter-medusa#104] [General]
Is this something thats planned?

Ideally, a company admin could set some default addresses. Members would see this in the checkout but could also choose from their own addresses if needed?

Would it be an option to simply get all addresses from the admin by their customer id? Maybe thats a smooth way to solve this without much overhead.

**Comment by fxmb** [comment]
Anyone?

**Comment by VariableVic** [comment]
This feature is on the backlog, not planned yet. Not sure yet what the implementation will look like. I think creating a new CompanyAddress model could make sense in a scenario where a company has multiple admins, or admin rights are transferred to a different employee.
